docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-dagster.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Deploy with Dagster
+description: How to deploy a pipeline with Dagster
+keywords: [how to, deploy a pipeline, Dagster]
+---
+
+# Deploy with Dagster
+
+
+## Introduction to Dagster
+
+Dagster is an orchestrator that's designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports. Dagster ensures these processes are reliable and focuses on using software-defined assets (SDAs) to simplify complex data management, enhance the ability to reuse code and provide a better understanding of data.
+
+To read more, please refer to Dagster’s [documentation.](https://docs.dagster.io/getting-started?_gl=1*19ikq9*_ga*NTMwNTUxNDAzLjE3MDg5Mjc4OTk.*_ga_84VRQZG7TV*MTcwOTkwNDY3MS4zLjEuMTcwOTkwNTYzNi41Ny4wLjA.*_gcl_au*OTM3OTU1ODMwLjE3MDg5Mjc5MDA.)
+
+### Dagster Cloud Features
+
+Dagster Cloud further enhances these features by providing an enterprise-level orchestration service with serverless or hybrid deployment options. It incorporates native branching and built-in CI/CD to prioritize the developer experience. It enables scalable, cost-effective operations without the hassle of infrastructure management. 
+
+### Dagster deployment options: **Serverless** versus **Hybrid**:
+
+The *serverless* option fully hosts the orchestration engine, while the *hybrid* model offers flexibility to use your computing resources, with Dagster managing the control plane. Reducing operational overhead and ensuring security.
+
+For more info, please [refer.](https://dagster.io/cloud)
+
+### Using Dagster for Free
+
+Dagster offers a 30-day free trial during which you can explore its features, such as pipeline orchestration, data quality checks, and embedded ELTs. You can try Dagster using its open source or by signing up for the trial. 
+
+## Building Data Pipelines with `dlt`
+
+`dlt` is an open-source Python library that allows you to declaratively load data sources into well-structured tables or datasets through automatic schema inference and evolution. It simplifies building data pipelines by providing functionality to support the entire extract and load process.
+
+How does `dlt` integrate with Dagster for pipeline orchestration?
+
+`dlt` integrates with Dagster for pipeline orchestration, providing a streamlined process for building, enhancing, and managing data pipelines. This enables developers to leverage `dlt`'s capabilities for handling data extraction and load and Dagster's orchestration features to efficiently manage and monitor data pipelines.
+
+Here’s a brief summary of how to orchestrate `dlt` pipeline on Dagster:
```

Comment:
We need to add some code snippets here. It feels like I read the post, but got nothing out of it. A simple dlt pipeline is orchestrated using Dagstser and then refers to more advanced stuff.

Reply:
See below for snippets

Reply:
added